get_bins.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+wget https://github.com/xen-troops/zephyr-dom0-xt/raw/rpi5_dom0_dev/prebuilt/zephyr_blinky.dtb -O zephyr_blinky.dtb
+wget https://github.com/xen-troops/zephyr-dom0-xt/raw/rpi5_dom0_dev/prebuilt/zephyr_sync.dtb -O zephyr_sync.dtb
+wget https://github.com/aoscloud/demo-services/raw/main/monkey_zephyr/src/helloworld_xen-arm64 -O helloworld_xen-arm64
```

Comment:
I think this script is better not to add here. We don't have any specific configuration in dom0, also - if we add it - we will need to support in 2 places. I think wiki page will work much better

Reply:
What two places? What configuration? What are you talking about? This is RPI5 release build which is expected to work out-of-the box. And everything is one place.

And if you have better solution - please provide it.